torch/_higher_order_ops/wrap.py

```diff
@@ -87,6 +87,43 @@ def wrapper():
 wrap_with_autocast = WrapWithAutocast()
 
 
+# This HOP allows you to bypass dynamo tracing of the wrapper function while
+# still tracing the inner function.
+# The wrapper function should receive a single callable argument, and return
```

Comment:
nit: something like "two callables, the first callable being the wrapper_fn which takes a single argument inner_fn.  The second callable is inner_fn and the rest of args and kwags are for the inner fn."